apps/ledger-live-mobile/ios/Podfile.lock

```diff
@@ -1938,7 +1938,7 @@ SPEC CHECKSUMS:
   Storyly: df48014753dfb30c663242c26478d5a6c58379e3
   storyly-react-native: 5c67a9cc9bf2a327ce6544a5093c35d8449ef727
   TOCropViewController: 20a14b6a7a098308bf369e7c8d700dc983a974e6
-  Yoga: 805bf71192903b20fc14babe48080582fee65a80
+  Yoga: d17d2cc8105eed528474683b42e2ea310e1daf61
```

Comment:
[ASK] Is this wanted ?

Reply:
I'm not sure, I would like to ask the question. I think this happens when running `pnpm mobile pod`.